src/_utils/itineraryCollapsible/ItineraryCollapsible.tsx

```diff
@@ -26,16 +27,23 @@ export default class ItineraryCollapsible extends PureComponent<ItineraryCollaps
     className: '',
     label: '',
     ariaLabel: '',
+    collapsed: true,
   }
 
   state: ItineraryCollapsibleState = {
-    collapsed: true,
+    collapsed: this.props.collapsed,
   }
 
   componentDidMount() {
     this.addListeners()
   }
 
+  componentDidUpdate(prevProps: ItineraryCollapsibleProps) {
+    if (prevProps.collapsed !== this.props.collapsed) {
+      this.setState({ collapsed: this.props.collapsed })
```

Comment:
In kairos we're blocked by eslint as it's considered a bad practice to update the state in a `componentDidUpdate` (https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-did-update-set-state.md)

Should you consider using `getDerivatedStateFromProps`?